daml-assistant/src/DAML/Assistant/Env.hs

```diff
@@ -42,6 +42,7 @@ getDamlEnv = do
     envProjectPath <- getProjectPath
     (envSdkVersion, envSdkPath) <- getSdk envDamlPath
         envDamlAssistantSdkVersion envProjectPath
+    envLatestStableSdkVersion <- fmap eitherToMaybe (try getLatestVersion :: IO (Either AssistantError SdkVersion))
```

Comment:
Is this a remote call to an http server? How often will it happen? Will  it work offline? Is it cached?

Reply:
Yes, every time the user runs `daml`. It will returns Nothing on failure, such as when the user is offline. It's not cached yet. How often should the cache be refreshed?

Reply:
I think if every command is doing it, we want to cache it now - otherwise when someone is benchmarking `daml compile ...` they're going to be benchmarking a remote HTTP request. It should be possible to configure the thing to never update unless you explicitly `daml update`. Otherwise daily seems fine?

Reply:
Yep, I will add caching now, and make the polling configurable soon.